generated/nidcpower/nidcpower/session.py

```diff
@@ -4851,6 +4852,100 @@ def fetch_multiple(self, count, timeout=hightime.timedelta(seconds=1.0)):
             )
         ]
 
+    @ivi_synchronized
+    def fetch_multiple_lcr(self, count, timeout=hightime.timedelta(seconds=1.0)):
+        '''fetch_multiple_lcr
+
+        Returns a list of previously measured LCRMeasurement instances on the specified channel that have been taken and stored in a buffer.
+
+        To use this method:
+
+        -  Set measure_when property to MeasureWhen.AUTOMATICALLY_AFTER_SOURCE_COMPLETE or MeasureWhen.ON_MEASURE_TRIGGER
+        -  Put the channel in the Running state (call initiate)
+
+        Note:
+        This method is not supported on all devices. For more information about supported devices, search ni.com for Supported Methods by Device.
+
+        Tip:
+        This method can be called on specific channels within your :py:class:`nidcpower.Session` instance.
+        Use Python index notation on the repeated capabilities container channels to specify a subset,
+        and then call this method on the result.
+
+        Example: :py:meth:`my_session.channels[ ... ].fetch_multiple_lcr`
+
+        To call the method on all channels, you can call it directly on the :py:class:`nidcpower.Session`.
+
+        Example: :py:meth:`my_session.fetch_multiple_lcr`
+
+        Args:
+            count (int): Specifies the number of measurements to fetch.
+
+            timeout (hightime.timedelta, datetime.timedelta, or float in seconds): Specifies the maximum time allowed for this method to complete, in seconds.
+                If the method does not complete within this time interval, NI-DCPower returns an error.
+                Default value: 1.0 second
+
+                Note:
+                When setting the timeout interval, ensure you take into account any triggers so that the timeout interval is long enough for your application.
+
+
+        Returns:
+            measurements (list of LCRMeasurement): A list of LCRMeasurement instances.
+
+                +-----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+                | channel               | The channel name associated with this LCR measurement.                                                                                                                                                |
```

Comment:
data type information is missing for these class members.

Reply:
I have updated the functions metadata to add a "type" column to the table in my latest commit.